raster/r.stream.extract/streams.c

```diff
@@ -598,6 +598,12 @@ int extract_streams(double threshold, double mont_exp, int internal_acc)
 	    
 	/* set main drainage direction to A* path if possible */
 	if (mfd_cells > 0 && max_side != np_side) {
+	    /* UNCERTAIN SOLUTION
+	    either:
+	    if (abs(wat_nbr[np_side] >= max_acc)) {
+	    or:
+	    if (fabs(wat_nbr[np_side]) >= max_acc) {
+	    */
 	    if (fabs(wat_nbr[np_side] >= max_acc)) {
```

Comment:
Please, see comment in code, which is the correct solution?

Reply:
The correct solution is the second suggestion: `if (fabs(wat_nbr[np_side]) >= max_acc) {`
However, this if block should be commented out as in [r.watershed](https://github.com/OSGeo/grass/blob/master/raster/r.watershed/seg/do_cum.c#L508).

Reply:
That would be my take also, re. logical test.

But comment out, why not delete?

Reply:
updated as suggested and commented out